### PR TITLE
Added Checklist API resource & 3 tests

### PR DESCRIPTION
**Pull request by mefemefe**
The current pipeline run is failing some Lists GUI tests but it's passing the tests added by this PR:

```automation    | ==============================================================================
automation    | Trello.Api.Trello Api Checklists Tests :: Example Test with Trello API Chec...
automation    | ==============================================================================
automation    | Verify A Checklist Can Be Created :: Verify that a checklist can b... | PASS |
automation    | ------------------------------------------------------------------------------
automation    | Verify A Checklist Can Be Deleted :: Verify that a checklist can b... | PASS |
automation    | ------------------------------------------------------------------------------
automation    | Verify A Checklist Can Be Updated :: Verify that a checklist's nam... | PASS |
automation    | ------------------------------------------------------------------------------
automation    | Trello.Api.Trello Api Checklists Tests :: Example Test with Trello... | PASS |
automation    | 3 tests, 3 passed, 0 failed

### CODE REVIEW

**Review by trab2 — APPROVED** [review]

**Comment by mefemefe** [comment]
Will wait until the new structure is merged to reopen this pr

**Review by nicolasvallejo13 — APPROVED** [review]

**Pull request merged**
